### PR TITLE
DSERV-22-docker-compose

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: docker-compose-index-m1.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        hard: 65536
+    cap_add:
+      - IPC_LOCK
+    volumes:
+      - "./config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml"
+      - "./config/log4j2.properties:/usr/share/elasticsearch/config/log4j2.properties"
+      - "./config/jvm.options:/usr/share/elasticsearch/config/jvm.options"
+      - ./esdata:/usr/share/elasticsearch/data
+    ports:
+      - 9201:9200
```
Comment (keenangraham) [inline docker-compose-index-m1.yml]: But why?

**Comment by keenangraham** [comment]
Screen Shot 2022-03-29 at 5 55 11 PM

I'm not sure this coverage step is actually doing anything.

**Comment by ottojolanki** [comment]
gah, must have broken coveralls with some of the cleanup things I did in the end.

File: docker-compose-index-m1.yml (hunk 1 above)
Comment (ottojolanki) [reply docker-compose-index-m1.yml]: That was to quickly take a peek at the ES while developing. Definitely not needed anymore. Will get rid of it and instead just expose the ports to the other services.

**Review by keenangraham — APPROVED** [review]

File: genomic_data_service/region_indexer.py [hunk 2]
```diff
@@ -7,13 +7,14 @@
 from os import environ
 from genomic_data_service.constants import FILE_HG19
 import argparse
+import time
```
Comment (mingjiecn) [inline genomic_data_service/region_indexer.py:10]: this import seems not used.

File: README.md [hunk 3]
```diff
@@ -6,60 +6,70 @@ Flask based web service providing genomic region search, based on regulomedb.org
...
+
+```bash
+docker-compose --file docker-compose-index-m1/intel.yml build
+docker-compose --file docker-compose-index-m1/intel.yml up
+```
+
+After indexing has finished (takes about 5 minutes) tear down:
+
+```bash
+docker-compose --file dockeri-compose-index-m1/intel.yml down --remove-orphans
```
Comment (mingjiecn) [inline README.md:38]: typo. dockeri should be docker

**Pull request merged**